src/api-gateway.ts

```diff
@@ -93,9 +98,9 @@ export class WatchApiGateway extends Construct {
       width: 12,
       stacked: false,
       left: [
-        this.createApiGatewayMetric(ApiGatewayMetric.Count, opts, { label: 'Calls', statistic: 'sum', color: '#1f77b4' }),
-        this.createApiGatewayMetric(ApiGatewayMetric.FourHundredError, opts, { label: 'HTTP 4XX', statistic: 'sum', color: '#ff7f0e' }),
-        this.createApiGatewayMetric(ApiGatewayMetric.FiveHundredError, opts, { label: 'HTTP 5XX', statistic: 'sum', color: '#d62728' }),
+        this.metrics.metricCalls(this.apiName, this.stage, opts),
+        this.metrics.metricErrors(this.apiName, this.stage, opts).errors4XX,
+        this.metrics.metricErrors(this.apiName, this.stage, opts).errors5XX,
```

Comment:
Is the removal of `color` optional intentional? Same question for the next block

Reply:
Kind of both, as I do not plan to customize colors very much; but I can keep it to increase backwards compatibility.